l10n/en/firefox/privacy/passwords.ftl

```diff
@@ -66,9 +66,9 @@ privacy-passwords-now-all-of = Now, all of this sounds pretty intimidating. The
 
 # Obsolete string
 #   $url_sumo-manager (string) - link to https://support.mozilla.org/kb/password-manager-remember-delete-edit-logins with additional attributes for analytics
-privacy-passwords-when-youre-logged = When you’re logged into { -brand-name-firefox } with your { -brand-name-firefox } account, you can sync across all your devices and access your passwords from a { -brand-name-firefox } mobile browser. Learn more about <a { $url_sumo_manager }>how to use the built-in password manager</a> to the fullest here.
+privacy-passwords-when-youre-logged = When you’re logged into { -brand-name-firefox } with your { -brand-name-firefox } account, you can sync across all your devices and access your passwords from a { -brand-name-firefox } mobile browser. Learn more about <a { $url_sumo-manager }>how to use the built-in password manager</a> to the fullest here.
 
-#   $url_sumo-manager (string) - link to https://support.mozilla.org/kb/password-manager-remember-delete-edit-logins with additional attributes for analytics
+#   $url_sumo_manager (string) - link to https://support.mozilla.org/kb/password-manager-remember-delete-edit-logins with additional attributes for analytics
```

Comment:
Ideally it'd be nice to standardise this so both strings are using `url_sumo_manager` and not two variants, but I get that means upstream changes in bedrock and potentially re-translation of exiting strings, so WFM!

Reply:
That's what the change did originally, but as per Flod's comments it makes no sense to change the original string which is now obsolete and uses the correct variable name.